Pod/Classes/RxRealm.swift

```diff
@@ -172,5 +172,36 @@ public extension Realm {
             }
         }
     }
-
-}
\ No newline at end of file
+    
+    /**
+     Returns bindable sink wich adds objects in sequence to Realm
+     */
+    public func rx_addObjects<S: SequenceType where S.Generator.Element: Object>() -> AnyObserver<S> {
+        return AnyObserver { [weak self] event in
+            switch event {
+            case .Next(let elements):
+                try! self?.write {
+                    self?.add(elements)
+                }
+            default:
+                break
+            }
+        }
+    }
+    
+    /**
+     Returns bindable sink wich adds object to Realm
+     */
+    public func rx_addObject<O: Object>() -> AnyObserver<O> {
+        return AnyObserver { [weak self] event in
+            switch event {
+            case .Next(let object):
+                try! self?.write {
+                    self?.add(object)
+                }
+            default:
+                break
+            }
+        }
+    }
+}
```

Comment:
one final though - do you think there's a way to have only `rx_add` that would automatically see if it gets one object or a sequence of objects as a parameter and do whatever it's proper? (just thinking out loud here)